_src/pattern-library/sass/components/_headings.scss

```diff
@@ -138,3 +138,9 @@ $headings-count: 8;
         @extend %hd-#{$i};
```

Comment:
There's [a ``.de-emphasized`` class](https://github.com/edx/ux-pattern-library/blob/frances/emphasized-extends/_src/pattern-library/sass/components/_headings.scss#L33) that's applied at the global level in this file too. Should we make an extend for that (even though it just has a color-based rule associated with it).